account_financial_report/wizard/general_ledger_wizard_view.xml

```diff
@@ -22,11 +22,11 @@
                             <field name="date_from" />
                             <field name="date_to" />
                             <field name="fy_start_date" invisible="1" />
+                            <field name="grouped_by" />
```

Comment:
Please keep this in the other column. You can move `target_move` to the first column that feels more natural.